engine/app/components/citizens_advice_components/sidebar.html.haml

```diff
@@ -0,0 +1,14 @@
+%aside.cads-sidebar
+  .cads-sidebar__content
+    %p.cads-sidebar__title= title
+    - if section_title.url.present?
```

Comment:
You can have this `if` inside the `h3`, that way you don't have to duplicate the `h3` element